next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   experimental: {
     logging: 'verbose',
+    isrMemoryCacheSize: 0, // disable ISR cache in memory and rely on file system cache.
```

Comment:
This is meant to be a fix for Kube containers but I figured it would help here too.
https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration#self-hosting-isr